tests/models/lvae/test_noise_model.py

```diff
@@ -106,9 +101,14 @@ def test_noise_model_likelihood(
 @pytest.mark.parametrize("img_size", [64, 128])
 @pytest.mark.parametrize("target_ch", [1, 3, 5])
 def test_multi_channel_noise_model_likelihood(
-    tmp_path: Path, n_gaussians: int, n_coeffs: int, img_size: int, target_ch: int
+    tmp_path: Path,
+    n_gaussians: int,
+    n_coeffs: int,
```

Comment:
I like the idea of moving `create_dummy_noise_model` to `pytest.fixture`. However, now it doesn't accept the parameters `n_gaussians` and `n_coeffs` anymore. 
Now, this is not a big issue as changing these parameters is not gonna break the code. Therefore, I am ok with not using them, but I'd like to have them removed completely from the code if we decide to go this way.